### DISCUSSION TITLE
为什么括号使用半角字符？

### DISCUSSION THREAD

**Discussion by Genteure** [vuejs-translations/docs-zh-cn#770] [Q&A] [answered]
> - 括号统一使用半角字符：`()`

https://github.com/vuejs-translations/docs-zh-cn/wiki/%E7%BF%BB%E8%AF%91%E9%A1%BB%E7%9F%A5#22-%E6%A0%87%E7%82%B9%E7%AC%A6%E5%8F%B7

我在本仓库以及 [vuejs/docs-next-zh-cn](https://github.com/vuejs/docs-next-zh-cn/) 的 issue 里搜索了一下，没有找到相关的讨论。

**Comment by Jinjiang** [comment, marked as answer]
是的，之前几个 Vue 版本的文档都是半角括号，一直是这样延续下来的
